clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -77,12 +78,15 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
                   Operation.newBuilder(operation.getId(), operation.getProducer())
                       .setFirst(operation.getFirst())
                       .setLast(operation.getLast())
-                      .build())
-              .setTimestamp(
-                  Instant.ofEpochSecond(
-                      auditLogRequest.getTimestamp().getSeconds(),
-                      auditLogRequest.getTimestamp().getNanos()))
-              .build();
+                      .build());
+
+      if (!(auditLogRequest.getTimestamp()).equals(Timestamp.getDefaultInstance())) {
```

Comment:
Is the default instance null?

Reply:
shoud use hasTimestamp() instead?

Reply:
default instance is not null. I tried to use null check but intellij indicates that timestamp will always not be null

Reply:
> shoud use hasTimestamp() instead?

I didn't know we have such method in Java protobuf timestamp.  This is more clear. Updated it.